{ [Error: Application cannot perform write actions. Contact Twitter Platform Operations through https://support.twitter.com/forms/platform]
  message: 'Application cannot perform write actions. Contact Twitter Platform Operations through https://support.twitter.com/forms/platform',
  statusCode: 403,
  twitterReply: '{"errors":[{"message":"Application cannot perform write actions. Contact Twitter Platform Operations through https://support.twitter.com/forms/platform","code":261}]}',
  code: 261,
  allErrors:
   [ { message: 'Application cannot perform write actions. Contact Twitter Platform Operations through https://support.twitter.com/forms/platform',
       code: 261 } ] }



TypeError: Cannot read property 'length' of undefined
    at /home/ubuntu/photo-mosaic-video-generator/peonbot.js:80:49
    at handleCallback (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/utils.js:95:12)
    at /home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/cursor.js:587:22
    at handleCallback (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:234:5)
    at /home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:542:22
    at queryCallback (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:167:20)
    at Callbacks.flush (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:86:7)
    at Server.destroy (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:694:41)
    at Server.close (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/server.js:345:17)
    at Db.close (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/db.js:273:19)


GivePicsLock taken by emptyDB()
collect: false
unresponded tweets: 29
picked doc
updated responded field to true in mongo

/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/utils.js:97
    process.nextTick(function() { throw err; });
                                        ^
TypeError: Cannot read property 'length' of undefined
    at /home/ubuntu/photo-mosaic-video-generator/peonbot.js:80:49
    at handleCallback (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/utils.js:95:12)
    at /home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/cursor.js:587:22
    at handleCallback (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:234:5)
    at /home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:542:22
    at queryCallback (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:167:20)
    at Callbacks.flush (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:86:7)
    at Server.destroy (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:694:41)
    at Server.close (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/server.js:345:17)
    at Db.close (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/db.js:273:19)
error: Forever detected script exited with code: 8




^C
ubuntu@ip-172-31-42-97:~/photo-mosaic-video-generator$ tail -f ~/.forever/r1Wk.log 
    at handleCallback (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/utils.js:95:12)
    at /home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/cursor.js:587:22
    at handleCallback (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:234:5)
    at /home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:542:22
    at queryCallback (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:167:20)
    at Callbacks.flush (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:86:7)
    at Server.destroy (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:694:41)
    at Server.close (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/server.js:345:17)
    at Db.close (/home/ubuntu/photo-mosaic-video-generator/node_modules/mongodb/lib/db.js:273:19)
error: Forever detected script exited with code: 8


186032049.png 1 10
givePics BLOCKED HERE by givePicsLock
givePics BLOCKED HERE by givePicsLock
stdout: Making mosaic of single image...
started: public/giveop144103186032049.png

stderr: Traceback (most recent call last):
  File "remoji.py", line 376, in <module>
    sys.exit(main())
  File "remoji.py", line 325, in main
    makeMosaic(targetImgName, scale, depth, littleImgs, outputName, colorMap, littleImgDir, origSkip = 1)
  File "remoji.py", line 164, in makeMosaic
    targetImg = Image.open(targetImgName).convert('RGB')
  File "/usr/local/lib/python2.7/dist-packages/PIL/Image.py", line 2248, in open
    fp = builtins.open(fp, "rb")
IOError: [Errno 2] No such file or directory: 'public/givePic144103186032049.png'

exec error: Error: Command failed: Traceback (most recent call last):
  File "remoji.py", line 376, in <module>
    sys.exit(main())
  File "remoji.py", line 325, in main
    makeMosaic(targetImgName, scale, depth, littleImgs, outputName, colorMap, littleImgDir, origSkip = 1)
  File "remoji.py", line 164, in makeMosaic
    targetImg = Image.open(targetImgName).convert('RGB')
  File "/usr/local/lib/python2.7/dist-packages/PIL/Image.py", line 2248, in open
    fp = builtins.open(fp, "rb")
IOError: [Errno 2] No such file or directory: 'public/givePic144103186032049.png'

./public/giveop144103186032049.png

fs.js:432
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory './public/giveop144103186032049.png'
    at Object.fs.openSync (fs.js:432:18)
    at Object.fs.readFileSync (fs.js:286:15)
    at Bot.picpost (/home/ubuntu/photo-mosaic-video-generator/peonbot.js:114:16)
    at /home/ubuntu/photo-mosaic-video-generator/peonbot.js:148:13
    at ChildProcess.exithandler (child_process.js:641:7)
    at ChildProcess.EventEmitter.emit (events.js:98:17)
    at maybeClose (child_process.js:743:16)
    at Process.ChildProcess._handle.onexit (child_process.js:810:5)




finished downloading pic
python remoji.py -s public/givePic144129886373164.png emoji/ public/giveop144129886373164.png 1 10
givePics BLOCKED HERE by givePicsLock
stdout: Making mosaic of single image...
started: public/giveop144129886373164.png
finished: public/giveop144129886373164.png
public/giveop144129886373164.png saved

stderr:
./public/giveop144129886373164.png
UNLOCKED givePicsLock inside of picpost()

/home/ubuntu/photo-mosaic-video-generator/peonbot.js:119
    var mediaIdStr = data.media_id_string;
                         ^
TypeError: Cannot read property 'media_id_string' of undefined
    at /home/ubuntu/photo-mosaic-video-generator/peonbot.js:119:26
    at responseHandler (/home/ubuntu/photo-mosaic-video-generator/node_modules/twit/lib/oarequest.js:362:14)
    at passBackControl (/home/ubuntu/photo-mosaic-video-generator/node_modules/twit/node_modules/oauth/lib/oauth.js:374:13)
    at IncomingMessage.<anonymous> (/home/ubuntu/photo-mosaic-video-generator/node_modules/twit/node_modules/oauth/lib/oauth.js:386:9)
    at IncomingMessage.EventEmitter.emit (events.js:117:20)
    at _stream_readable.js:920:16
    at process._tickCallback (node.js:415:13)

